config.json

```diff
@@ -177,6 +177,7 @@
     "checkRadio-color": "#4f4f4f",
     "checkRadio-backgroundColor": "#ffffff",
     "checkRadio-borderColor": "#dfdfdf",
+    "sweetAlert-backgroundColor": "#ffffff",
```

Comment:
Can we use a more generic config name? This name ties to a specific vendor library. Maybe just `alert-backgroundColor`.